integration/src/main/java/scripts/testscripts/ControlledGcsBucketLifecycle.java

```diff
@@ -115,24 +101,7 @@ public void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspace
     ControlledGcpResourceApi resourceApi =
         ClientTestUtils.getControlledGcpResourceClient(testUser, server);
 
-    // Create a bucket with a name that's already taken in another project. This should fail, as
-    // bucket names are globally unique in GCP.
-    ApiException duplicateNameFails =
-        assertThrows(
-            ApiException.class,
-            () -> createBucketAttempt(resourceApi, existingBucket.getBucketName()));
-    assertEquals(HttpStatus.SC_CONFLICT, duplicateNameFails.getCode());
-    logger.info("Failed to create bucket with duplicate name, as expected");
-
-    // Create a bucket with a name that's already taken by a publicly accessible bucket. WSM should
-    // have get and read access, as the bucket is open to everyone, but this should still fail.
-    ApiException publicDuplicateNameFails =
-        assertThrows(
-            ApiException.class, () -> createBucketAttempt(resourceApi, PUBLIC_GCP_BUCKET_NAME));
-    assertEquals(HttpStatus.SC_CONFLICT, publicDuplicateNameFails.getCode());
-    logger.info("Failed to create bucket with duplicate name of public bucket, as expected");
```

Comment:
I think we should keep this test. There's logic inside the create bucket step checking whether a bucket already exists and if so, whether it's in the current workspace project (i.e. the Stairway step is re-running which is fine) or not (which is a 409 error).

We previously used read access as a proxy for that, and WSM would let users add public buckets as "controlled" resources (which then caused problems when you deleted them). I'd like to keep testing that we can't do this again